src/pages/docs/pages/server-rendering.md

```diff
@@ -24,7 +24,7 @@ The above would serve content in a browser at the path _/users/_.
 
 In your page file, Greenwood supports the following functions that you can `export` for providing server rendered content and [frontmatter](/docs/resources/markdown/) to produce the `<body><body>` content for your page.
 
-- **default** (recommended): Use the custom elements API to render out your page content, aka **Web (Server) Components**. _This will take precedence over `getBody`_ and will also automatically track your custom element dependencies (in place of having to define frontmatter imports in `getFrontmatter`).
+- **default** (recommended): Use the custom elements API to render out your page content, aka **Web (Server) Components**. _This will take precedence over `getBody`_ and will also automatically track your custom element dependencies (in place of having to define frontmatter imports in `getFrontmatter`). However, any custom elements tracked by this automatic dependency management _must_ be able to render on the server. Any elements that require the browser should be imported using the imports field in `getFrontmatter`.
```

Comment:
Yeah, I guess it was assumed that since this is an SSR context, SSR compatibility would also be assumed, but I guess better to be explicit I suppose.

I would say you can also include things client side by adding `<script` and `<link>`  tags in `getLayout` as well.

Reply:
Actually, let me rethink this since I think the automatic dependency tracking may not actually be a feature here, since SSR pages only execute on the server and don't ship any JS to the client, which may have also contributed to the confusion here.